src/utilities.ts

```diff
@@ -53,4 +55,6 @@ export async function execCommand(toolPath: string, args: string[], options: Exe
     }
 
     return execResult;
-}
\ No newline at end of file
+}
+
+export const latest = 'latest';
```

Comment:
small nitpick but constants like this are usually capitalized like LATEST. When it's something that is used periodically and is a constant value.